src/Angor/Avalonia/Angor.UI.Model.Implementation/Wallet/TransactionPreview.cs

```diff
@@ -0,0 +1,19 @@
+using Angor.Wallet.Application;
+using Angor.Wallet.Domain;
+using CSharpFunctionalExtensions;
+
+namespace Angor.UI.Model.Implementation.Wallet;
+
+public class TransactionPreview(WalletId walletId, long amount, string address, long feeRate, Fee fee, IWalletAppService walletAppService) : ITransactionPreview
```

Comment:
This is weird, I can still see it as transaction Preview

Reply:
Somewhat, I forgot to push the commits...